app/routes/lending/components/LendableObjectEditor.tsx

```diff
@@ -82,6 +84,12 @@ export const LendableObjectEditor = ({ initialValues }: Props) => {
             placeholder="Grill"
             component={TextInput.Field}
           />
+          <Field
+            label="Kontakt e-post"
```

Comment:
```suggestion
            label="E-post til kontaktperson"
```
Wouldn't the correct norwegian be "Kontakte-post" or something?